lib/beautiful/xresources.lua

```diff
@@ -70,13 +70,21 @@ local function get_screen(s)
     return s and screen[s]
 end
 
---- Get global or per-screen DPI value falling back to xrdb.
--- @tparam[opt] integer|screen s The screen.
+--- Get global or per-screen DPI value, falling back to xrdb.
+-- @screen[opt] s The screen.
 -- @treturn number DPI value.
 function xresources.get_dpi(s)
-    s = get_screen(s)
-    if dpi_per_screen[s] then
-        return dpi_per_screen[s]
+    if s then
```

Comment:
Well, previously calling with `nil` means that `get_screen` returned `nil` and then `dpi_per_screen[nil]` would cause a Lua error "table index is nil".
Why do you want to distinguish between the argument and the return value?